do_cmake.sh

```diff
@@ -57,7 +57,7 @@ if type cmake3 > /dev/null 2>&1 ; then
 else
     CMAKE=cmake
 fi
-${CMAKE} -DCMAKE_BUILD_TYPE=Debug $ARGS "$@" .. || exit 1
+${CMAKE} $ARGS "$@" .. || exit 1
```

Comment:
I'm afraid that outcome of  this change may be surprising to many people. Over the time developers might got used to and expect presence of the debug stuff in default-configured builds.
Can we satisfy the requirement for crimson perf testing in another way? If not, then we should consider sending prior notification to ceph-devel or, at least, print message signalizing the change at the end of `do_cmake.sh`, I think.

Reply:
mail sent.

Reply:
with #30799, i think this change is backward compatible

Reply:
There shouldn't be a difference for the `perf test` bot as we already pass `-DCMAKE_BUILD_TYPE=Release` there.

Reply:
i see, didn't realize that cmake allows us to pass multiple `-D<FOO>=<BAR>` with the same `<FOO>`, and the last wins.